### PR TITLE
Add tests for "codecoverage-tools" package: "utilities.ts" and "codecoverageutilities.ts"

### PR DESCRIPTION
**Pull request by sergey-koryshev**
**Description**
Added Unit Tests for package `codecoverage-tools` for files `utilities.ts` and `codecoverageutilities.ts`

**Screenshot**
image

### CODE REVIEW

**Review by ismayilov-ismayil — APPROVED** [review]

File: common-npm-packages/codecoverage-tools/Tests/data/fakeData.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import * as cheerio from 'cheerio';
```
Comment (KonstantinTyukalov) [inline common-npm-packages/codecoverage-tools/Tests/data/fakeData.ts:1]: Is it used somewhere?

**Review by KonstantinTyukalov — APPROVED** [review]

**Review by vmapetr — APPROVED** [review]

File: common-npm-packages/codecoverage-tools/Tests/data/fakeData.ts (hunk 1 above)
Comment (sergey-koryshev) [reply common-npm-packages/codecoverage-tools/Tests/data/fakeData.ts]: Removed it for now. It will be used in next suites

**Review by Pranshu-Negi — APPROVED** [review]

**Pull request merged**